recipes/rsatoolbox/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "rsatoolbox" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rsatoolbox-{{ version }}.tar.gz
+  sha256: 2d091cbaa33373bf9da4df5ca8d127f0e427431a3db726076090ab2d54fe1213
+
+build:
+  skip: true  # [py<37 or py>310 or py2k]
```

Comment:
```suggestion
```

None of these are necessary: python 2 & <3.7 aren't built in conda-forge anymore, and 3.11 will only be added by the migrator (which can then be deferred until there's a rsatoolbox version that supports it).

Reply:
Got it. Removed the skip entry.